js/functions.js

```diff
@@ -172,6 +172,23 @@ function generateIcons( data ) {
 
          
     // }
+    function fillBarAnimation (){
+        var i,
+        h = window.innerHeight - (elementHeight('#skills')/2),
+        scroll = window.scrollY + h,
+        bar = document.querySelectorAll('.progress-block > .bar > .bar-value > .bar-fill'),
+        barTop = document.getElementById('skills').offsetTop;
+    
+        for(i = 0; i < bar.length; i++ ){
```

Comment:
Vietoje `for` ciklo, būtų galima pabandyti `forEach` cikla, kodas būtų šiek tiek elegantiškesnis, nors šiaip didesnio skirtumo tai neduotų.